backend/repository/handler.go

```diff
@@ -347,6 +348,45 @@ func (h *Handler) CreateDiff(w http.ResponseWriter, r *http.Request, ps httprout
 	h.WriteJSON(w, http.StatusOK, &createDiffResponse{DiffURI: resp})
 }
 
+// LandDiff lands a diff.
+func (h *Handler) LandDiff(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	type LandDiffRequest struct {
```

Comment:
I prefer defining type inside a func if the struct type is only to be used within the func, similar to a func scope variable. Parsing the request is a one-time thing. If we have multiple handlers like this, defining a type inside a func seems to be more readable.